.github/pull_request_template.md

```diff
@@ -1 +1,96 @@
+
+Please update the PR's name to include a quick note about every linked PR.
+
+## How has the API changed?
+
+Please paste the output of `elm diff` run on latest master in the code block:
+
+```
+
+```
+
+## Releasing
+
+After this PR merges, and you've pulled down latest master, finish following the [publishing process](https://github.com/NoRedInk/noredink-ui/blob/master/README.md#publishing-a-new-version).
+
+
+---
+
+# :star2: Adding a new component
+
+## About the component
+
+What is the purpose of the new component? When and where will it be used? How will the reviewer know if this component meets the success criteria?
+
+Please link to any relevant context and stories.
+
+
+## :framed_picture: What does this change look like?
+
+- Fill in with screenshots and ping design.
+- [Guidelines for an optimal design ping](https://paper.dropbox.com/doc/Guidelines-for-Sharing-User-Facing-Changes-with-Design--BpL8hpJLMugy6033aT5m0JdaAg-bdKGQtYH9qO9I00hUkA6k)
+
+
+## Component completion checklist
+
+- [ ] Component has clear documentation
+- [ ] Component is in the component library example app
+    - [ ] Component is categorized reasonably (see [Category](https://github.com/NoRedInk/noredink-ui/blob/master/styleguide-app/Category.elm) for all the currently available categories). The component can be in multiple categories, if appropriate.
+    - [ ] Component has a cute preview for the component library example cards
+    - [ ] Component has a customizable example. Aim for having _every_ possible supported version of the component displayable through the configuration on this page.
+    - [ ] The customizable example produces sample code
+    - [ ] The component's example page includes keyboard behavior, if any
+    - [ ] The component's example page includes guidance around how to use the component, if necessary
+- [ ] Component is tested
+    - axe checks and percy snapshots will be automatically added for every component. However, if you want to customize _when_ the checks and snapshots are made, you will need to make changes to [script/puppeteer-tests.js](https://github.com/NoRedInk/noredink-ui/blob/master/script/puppeteer-tests.js).
+    - there are 2 ways to add Elm tests:
+        - if you want to test the styleguide example directly, add ProgramTest-style tests under `styleguide/tests`
+        - if you want to test the component directly, add tests under `tests/Spec`. Historically, this has been the more popular Elm testing strategy for noredink-ui.
+- [ ] Component API follows standard patterns in noredink-ui
+    - e.g., as a dev, I can conveniently add an `nriDescription`
+    - and adding a new feature to the component will _not_ require major API changes to the comopnent
+- [ ] I've gone through the relevant sections of the [Development Accessibility guide](https://paper.dropbox.com/doc/Accessibility-guide-4-Development--BiIVdijSaoijjOuhz3iTCJJ1Ag-rGoHpC91pFg3zTrYpvOCQ) with this component in mind
```

Comment:
```suggestion
- [ ] I've gone through the relevant sections of the [Development Accessibility guide](https://paper.dropbox.com/doc/Accessibility-guide-4-Development--BiIVdijSaoijjOuhz3iTCJJ1Ag-rGoHpC91pFg3zTrYpvOCQ) with this component in mind
```

I think I prefer this checkbox to be the first in this section so they might take a look upfront instead of waiting until later. wdyt?